### ISSUE TITLE
Power profiler connection

### ISSUE THREAD

**Issue by yannisarnouts** [MOSAIC-LoPoW/octa-stack-students#34]
Hi

How can I connect the power profiler with the Octa?

**Comment by Klievan** [comment]
There's two ways, one which uses an nRF52 devkit and one which does not but requires a JLINK programmer.
https://infocenter.nordicsemi.com/topic/ug_ppk/UG/ppk/PPK_user_guide_PPK_on_customHW_with_nRF5xDK.html
https://infocenter.nordicsemi.com/topic/ug_ppk/UG/ppk/PPK_user_guide_PPK_on_customHW_standalone.html

I THINK you can just connect to the OCTA through VDD/VIN and GND, no @MatsDeMeyer ?

**Comment by MatsDeMeyer** [comment]
Yes, correct! You can supply 3.3V directly to the octa.

**Comment by yannisarnouts** [comment]
![afbeelding](https://user-images.githubusercontent.com/34567165/144437830-a541fe1f-78fe-425b-bb60-7eb10504c5df.png)

Okay so just like this? It either disconnects the whole time when the PPK is opened.
Or I get this error message.
![afbeelding](https://user-images.githubusercontent.com/34567165/144437953-606330af-8639-4374-b1d1-fa6ebaa2fed4.png)

**Comment by yannisarnouts** [comment]
So just like this?
![afbeelding](https://user-images.githubusercontent.com/34567165/144441323-c20ec15e-0e0d-4ad1-a297-e480718f518c.png)
It connects, but every time this happens and connection is lost
![afbeelding](https://user-images.githubusercontent.com/34567165/144441404-93dbcf19-87ea-4cd8-a1c0-f8bed363ccd0.png)

**Comment by Klievan** [comment]
Hi Yannis. Try reviewing your setup. I don't see a devkit beneath your powerprofiler, or a JLINK-debugger attached. Your options:
1. Place your power profiler on your BLE devkit, and switch the USB from your profiler onto the main devkit USB-port. 
2. Use a JLINK-debugger and connect it to your powerprofiling kit.
Make sure you follow the instructions on either two URL's I've sent you. It explains on how to set the switches, and where to connect your USB.

**Comment by yannisarnouts** [comment]
Yes the devkit is beneath the power profiler. Not really clear in the photo indeed ☺️

**Comment by Klievan** [comment]
Allright my bad! Have you connected the USB to the USB of the devkit?

**Comment by yannisarnouts** [comment]
Devkit

**Comment by Klievan** [comment]
What happens if you disconnect the jumpers from the PPK and try to measure "nothing"?

**Comment by yannisarnouts** [comment]
Okay so DUT select is set to External and power select to Reg.

First time: it starts but I get an error, because there is no connection.

![afbeelding](https://user-images.githubusercontent.com/34567165/144448984-755c6888-77d6-4c2e-adaa-0e00f8510c51.png)

I try to connect again, now it works and measures
![afbeelding](https://user-images.githubusercontent.com/34567165/144451652-35d9722a-47cc-4b76-813a-16d49ef0fa72.png)

**Comment by Klievan** [comment]
While you are measuring, try connecting the device. Does the power profiling kit disconnect when you attach the jumper cables or do you get a normal reading?

**Comment by yannisarnouts** [comment]
the led between the 2 buttons lights (on the octa) for a second, but immediately disconnects

**Comment by MatsDeMeyer** [comment]
I'm afraid the startup current is too high in that case. What happens if you take off the GPS and murata module?

**Comment by yannisarnouts** [comment]
Yes it works now, afterwards I can reconnect the modules.
Thanks @Klievan and @MatsDeMeyer 🥳 🥇

**Comment by MatsDeMeyer** [comment]
You can also neutralize this startup peak by connecting a battery, and disconnecting this after a few seconds.
